docs/docsite/rst/community/contributor_path.rst

```diff
@@ -0,0 +1,104 @@
+****************
+Contributor path
+****************
+
+This section describes the contributor journey from the early beginning, through all the stages to becoming a leader who helps shape the future of Ansible. You can use this path as a roadmap for your long-term participation.
+
+Any contribution to the project, even a small one, is very welcome and valuable. Any contribution counts, whether it's feedback on an isssue, pull requiest, or  topic, a documentation change, or a coding contribution. When you contribute regularly, your proficiency and judgment in the related area are increasing fast and, along with this, the importance of your presence in the project.
+
+.. contents::
+    :local:
+
+
+
+Determine your area of interest
+=================================
+
+First, determine areas that are interesting to you, taking into consideration your current experience and experience you'd like to gain. For example, if you use a certain collection, have a look there. See :ref:`how_can_i_help` for more ideas on how to help.
+
+Find the corresponding the project
+====================================
+
+These are multiple community projects in the Ansible ecosystem you could contribute to:
+
+- `Ansible Core <https://docs.ansible.com/ansible-core/devel/index.html>`_
+- `Collections <https://docs.ansible.com/ansible/latest/user_guide/collections_using.html>`_
+- `AWX <https://github.com/ansible/awx>`_
+- `Galaxy <https://galaxy.ansible.com/>`_
+- `ansible-lint <https://ansible-lint.readthedocs.io/en/latest/>`_
+- `Molecule <https://molecule.readthedocs.io/en/latest/>`_
+
+Learn
+=====
+
+The required skillset depends on the area of interest and the project you'll be working on. Remember that the best way to learn is by doing.
```

Comment:
"Remember that the best way to learn is by doing."
is this line complete?

Reply:
Looks complete to me, i.e. "Keep it in mind that the best way to learn is by doing."